firmware/tunerstudio/tunerstudio.template.ini

```diff
@@ -3704,6 +3704,11 @@ cmd_set_engine_type_default					= "@@TS_IO_TEST_COMMAND_char@@@@ts_command_e_TS_
 			settingOption = "Toyota C60"                 , totalGearsCount=5,gearRatio1=3.17,gearRatio2=2.05,gearRatio3=1.48,gearRatio4=1.17,gearRatio5=0.92,gearRatio6=0.73     ,finalGearRatio=4.53
 			settingOption = "Toyota EB60"                , totalGearsCount=6,gearRatio1=3.54,gearRatio2=2.05,gearRatio3=1.38,gearRatio4=1.03,gearRatio5=0.88,gearRatio6=0.73     ,finalGearRatio=4.06
 			settingOption = "Toyota EB62"                , totalGearsCount=6,gearRatio1=3.54,gearRatio2=2.05,gearRatio3=1.38,gearRatio4=0.98,gearRatio5=0.74,gearRatio6=0.66     ,finalGearRatio=4.06
+			settingOption = "Aisin AR5 5 speed"          , totalGearsCount=5,gearRatio1=3.750,gearRatio2=2.260,gearRatio3=1.510,gearRatio4=1.000,gearRatio5=0.730
```

Comment:
Round to 2 digits - the fields are stored in multiples of 0.01, so while this will kind of work, TS gets a little mad about it.